### DISCUSSION TITLE
Nextcloud Office --- Unable to create a new file from a template

### DISCUSSION THREAD

**Discussion by Menzaah** [nextcloud/all-in-one#2345] [Questions] [answered]
### Steps to reproduce
1. Create an Excel table template
2. Import an Excel template into Nextcloud Office settings
3. Try creating a new spreadsheet from this template

### Expected behavior 

The spreadsheet is supposed to open and therefore the file is created.

### Actual behavior 

Via the Nextcloud application it is impossible to create a new file from the templates. On Android it is downright impossible to select a template and on iOS during creation it runs in a vacuum and ends up creating an empty file in the wrong format.
On the other hand, via the web interface from a computer, this poses no problem.

### Host OS 

Ubuntu 20.04

#### Nextcloud AIO version 

Nextcloud AIO v4.8.1

#### Current channel 

Latest channel

#### Other valuable info 

On creation failures via iOS, no error log is generated and no error messages on the device. On the android side I can't even create a file from a template.
Here are the only logs I have regarding Richdocuments that I don't understand the origin at all.

[richdocuments-error.txt](https://github.com/nextcloud/all-in-one/files/11239209/richdocuments-error.txt)
![Logs](https://user-images.githubusercontent.com/78805485/232210760-0e139da7-9127-427c-8929-3ae48361006a.PNG)

Thank you in advance for all the help provided, it's not totally blocking but quite disabling.

**Comment by szaimen** [comment, marked as answer]
See https://github.com/nextcloud/richdocuments/issues/2918

**Reply by Menzaah** [reply]
Sorry, I'm new to GitHub, I wasn't sure I opened a topic in the right place.
On this subject, I think I opened the subject in the wrong place, and I therefore created in the "nextcloud - AIO" section of the Github , here : https://github.com/nextcloud/all-in-one/issues/2346
